style.css

```diff
@@ -36,8 +36,65 @@ body {
 }
 
 .menu {
-  align-self: center;
-  margin-left: 249px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background-color: transparent;
+  border: none;
+  font-size: 20px;
+  color: #67798e;
+  width: 100%;
+  padding-right: 20px;
+  margin-top: 8px;
+}
+
+.mobile-menu {
+  background-color: #fff;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 3;
+}
```

Comment:
### Missing Section in your menu
![Figma-Design-Missing-Section](https://user-images.githubusercontent.com/93039359/196635759-dd3cbce3-1d0f-4868-b9fb-b46f13b63d3b.PNG)

- [ ] Your menu has this missing section which you can see in the Figma template. Kindly add it to make your design consistent with the Figma template and also to make your project professional 😊

Reply:
Good day @HAMMAS-SALEEM and Good day to my new code reviewer, thank you for the feedback I really do appreciate it but could you please kindly take note that the required changes ( the bottom mobile nav bar ) are mobile mockup designs, meaning that the bar mentioned is used for iPhone devices and is not part of the Figma design as it would create a design conflict for android users as the bar represents the slide up nav/menu that is on the majority of the latest iPhone devices where as android devices have 3 nav buttons (view open tabs, home and return/back) in place of the iPhone nav bar, the same principles and reasoning behind why we do not add the time sign, mobile data signal sign, wifi signal sign, and the battery status sign that is located in the circled top section is the same reasoning behind why we do not add the slide up nav bar as the users devices already display their own UI that apply to the Figma design,

![bandicam 2022-10-19 11-17-38-472](https://user-images.githubusercontent.com/90766123/196654230-bf93c30f-985e-4b56-8f3d-6755fcc2d289.jpg)